ord_schema/templating.py

```diff
@@ -63,20 +63,65 @@ def _escape(string):
     return string.replace('\\', '\\\\')
 
 
+def _is_null(value):
+    """Returns whether a value is null."""
+    return pd.isnull(value) or value == 'nan'
+
+
 def _replace(string, substitutions):
```

Comment:
Could you rename this function? It now performs string substitutions, parses the reaction, performs the empty field checks, and returns a Reaction message rather than a string. I feel like "_replace" doesn't do it justice.

Reply:
Changed to `_fill_template`.